metrics/sender/log.go

```diff
@@ -0,0 +1,45 @@
+package sender
+
+import (
+	"io"
+	"log"
+	"os"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/pkg/errors"
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+const Log = "log"
```

Comment:
__[golint]__
* [golint] exported const Log should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->